# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1252013046 14400
#      Thu Sep 03 17:24:06 2009 -0400
# Node ID bb36a5730ecc57532ba4e13ded666149306ace41
# Parent  599e92aaa4c083108968862ac12ab94838098cf0
configure.in (AH_BOTTOM): If HAVE_Z is defined, define HAVE_ZLIB

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2009-09-03  John W. Eaton  <jwe@octave.org>
+
+	* configure.in (AH_BOTTOM): If HAVE_Z is defined, define HAVE_ZLIB.
+
 2009-09-03  Jaroslav Hajek  <highegg@gmail.com>
 
 	* NEWS: Update.
 
 2009-09-02  Jaroslav Hajek  <highegg@gmail.com>
 
 	* configure.in (FLOAT_TRUNCATE): New config macro.
 
diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -2023,16 +2023,21 @@ typedef int sig_atomic_t;
 #define OCTAVE_EMPTY_CPP_ARG
 
 // Octave is currently unable to use FFTW unless both float
 // and double versions are both available.
 #if defined (HAVE_FFTW3) && defined (HAVE_FFTW3F)
 #define HAVE_FFTW
 #endif
 
+// Backward compatibility.
+#if defined (HAVE_Z)
+#define HAVE_ZLIB
+#endif
+
 #include "oct-dlldefs.h"
 #include "oct-types.h"
 ])
 
 ### Do the substitutions in all the Makefiles.
 
 AC_CONFIG_COMMANDS([Makefile], [if test "$ac_srcdir" != "."; then
   cp $srcdir/Makefile .
